tests/create_task_ignore_error.yml

```diff
@@ -0,0 +1,39 @@
+name: Create Tes Task Job With Ignore Error Flag
+description: Job to create a new TES Task with ignore_error flag
+service: TES
+compatible_versions:
+  - v1.1
+tags:
+  - logical
+  - create task
+  - create task test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "CompTest",
+        "description": "CompTest",
+        "executors": [
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "error"
+            ],
+            "ignore_error": true
```

Comment:
If this were a functional test (or if there were a functional variant of this test), it would be good to have a negative test for this as well. That is, the same test with `ignore_error` set to `false` and expecting the task to fail.